i18n/en.json

```diff
@@ -204,7 +204,7 @@
 	"smw-ask-sorting": "Sorting",
 	"smw-ask-search": "Search",
 	"smw-ask-format-selection-help": "For a detailed description, please visit the $1 help page.",
-	"smw-ask-query-search-info":"The query <code><nowiki>$1</nowiki></code> was answered by the {{PLURAL:$3|<code>$2</code> (from cache)|<code>$2</code>}} in $4 {{PLURAL:$4|second|seconds}}.",
+	"smw-ask-query-search-info":"The query <code><nowiki>$1</nowiki></code> was answered by the {{PLURAL:$3|<code>$2</code>|<code>$2</code> (from cache)}} in $4 {{PLURAL:$4|second|seconds}}.",
```

Comment:
From what I have read this should be it (assuming that true is singular = from cache):

```The query <code><nowiki>$1</nowiki></code> was answered by the {{PLURAL:$3|1=<code>$2</code> (from cache)|<code>$2</code> (from cache)|<code>$2</code>}} in $4 {{PLURAL:$4|second|seconds}}```

@Nikerabbit If you could send an ok and I will add this ...